exporters/otlp/src/otlp_http_client.cc

```diff
@@ -63,6 +51,146 @@ namespace otlp
 namespace
 {
 
+using Base64EscapeChars                              = const unsigned char[64];
```

Comment:
should these methods be moved to https://github.com/open-telemetry/opentelemetry-cpp/blob/main/api/include/opentelemetry/common/string_util.h ?

Reply:
There are a lot of codes here, do you think it's better to move it into sdk?

Reply:
Yeah good point, not required. We can move it later if need to share with other components. Thanks.

Reply:
base64 algorithm is moved into sdk now.

Reply:
Really sorry if my comment above was not clear. I did realize that this will bloat the sdk, and we should move it only when we need to share it with other components.